### PR TITLE
docs: SSH options design

### PR DESCRIPTION
**Pull request by pvoborni**
Adding a design for configuring ssh options for mrack's ssh connection
check and Ansible inventory output with main purpose to have the Ansible
inventory settings consumable by Ansible calls used in  pytest-ansible.

Signed-off-by: Petr Vobornik <pvoborni@redhat.com>

### CODE REVIEW

File: docs/designs/ssh_options.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- `-o 'StrictHostKeyChecking=no'` for accepting unknown hosts
+- `-o 'UserKnownHostsFile=/dev/null'` for not remembering the hosts and thus
+  being able to destroy them, recreate new with same hostname and still work.
+- `-o 'ServerAliveInterval=60'`, `-o 'ServerAliveCountMax=5'` for more resilient
+  SSH connection
+
+
+Mrack is already using `StrictHostKeyChecking` and `UserKnownHostsFile` in its
+ssh connection check, but they are not part of inventory and thus an SSH connection
+from Ansible, if similar options are not provided via other mean, will fail.
```
Comment (XiyangDong) [inline docs/designs/ssh_options.md:21]: via other ways/by other means ?

**Comment by XiyangDong** [comment]
The context of this design looks good to me and well written. Approving

**Review by XiyangDong — APPROVED** [review]

**Review by Tiboris — APPROVED** [review]
Thanks @pvoborni

File: docs/designs/ssh_options.md (hunk 1 above)
Comment (pvoborni) [reply docs/designs/ssh_options.md]: Fixed

**Pull request merged**